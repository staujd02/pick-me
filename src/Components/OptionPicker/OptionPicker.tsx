import React from 'react';

export type OptionPickerProps = {
  finishedPickingOptionsCallback: (chosenOptions: Option[]) => void
};
export type Option = {
  image: string
  title: string
}

const OptionPicker: React.FC<OptionPickerProps> = (props: OptionPickerProps) => {
  return (
    <div className="option-picker"></div>
  );
}

export default OptionPicker;
