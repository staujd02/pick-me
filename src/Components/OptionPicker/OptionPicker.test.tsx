import React from 'react';
import { shallow, ShallowWrapper } from "enzyme";
import { formatHTML } from '../../TestUtilities/htmlFormatter';
import OptionPicker from './OptionPicker';

describe('The App', () => {

  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<OptionPicker />);
  });

  it('renders correctly', () => expect(formatHTML(wrapper.html())).toMatchSnapshot());
});