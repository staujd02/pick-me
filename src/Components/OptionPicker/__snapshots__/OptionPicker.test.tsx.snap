// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App renders correctly 1`] = `
"
<div class=\\"option-picker\\"></div>
"
`;
